src/obb/server/routes.clj

```diff
@@ -0,0 +1,13 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.routes
+  "Specifices the routes and associates handlers"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
+    [compojure.route :as route]
+    [obb.server.handlers.index :as index]))
+
+(compojure/defroutes public-routes
+  "The routes available to be served, that don't need auth"
+  (GET "/" request (index/handler request))
+  (route/not-found "No such page."))
```

Comment:
Why did you feel the need to add the message `No such page.`? I think in this case it is not needed. `HTTP 404 Not Found` conveys exactly that, right?

I looked at the Compojure source and found it strange that the param for `route/not-found` is required. :s